lib/cn/csched_sp3.c

```diff
@@ -1125,16 +1121,6 @@ sp3_process_workitem(struct sp3 *sp, struct cn_compaction_work *w)
     if (w->cw_debug & (CW_DEBUG_PROGRESS | CW_DEBUG_FINAL))
         sp3_log_progress(w, &w->cw_stats, true);
 
-    if (cn_node_isroot(tn)) {
-        u64 dt;
-
-        /* Maintain an average of the root spill's build time - used for throttling.
-         */
-        dt = w->cw_t3_build - w->cw_t2_prep;
-        sp->rspill_dt_prev = (dt + sp->rspill_dt_prev) / 2;
-        atomic_set(&sp->rspill_dt, sp->rspill_dt_prev);
-    }
-
```

Comment:
Keeping an average rspill time for the entire tree was problematic becuase it allowed trees with tiny/fast spills to dominate the throttle time (by relaxing it too much), thereby causing large/slow spills to get further behind since ingest isn't being sufficiently throttled.